ui/packages/atlasmap/src/UI/MappingTransformation.tsx

```diff
@@ -46,6 +46,7 @@ export interface IMappingTransformationProps {
   name: string;
   transformationsOptions: ITransformationSelectOption[];
   transformationsArguments?: ITransformationArgument[];
+  mappingExpressionEnabled: boolean;
```

Comment:
I suggest to change this to be something like `disableTransformations` to reflect what needs to be done instead of why.

Reply:
I'm happy to change the name but I think it's important to retain that the multiplicity transformation enablement is tied to the mapping expression enablement.  so:
`isDisabled={mappingExpressionEnabled}` maks that clar in the code.

Reply:
Yes that’s important indeed but the information will be retained on the outermost layer, the `MappingDetailsView`, where the `disableTransformation` prop is initially set